chain/x/zoracle/client/rest/query.go

```diff
@@ -25,3 +27,36 @@ func getRequestHandler(cliCtx context.CLIContext, storeName string) http.Handler
 		rest.PostProcessResponse(w, cliCtx, res)
 	}
 }
+
+func getScriptHandler(cliCtx context.CLIContext, storeName string) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		vars := mux.Vars(r)
+		codeHash := vars[codeHash]
+
+		res, _, err := cliCtx.QueryWithData(fmt.Sprintf("custom/%s/script/%s", storeName, codeHash), nil)
+		if err != nil {
+			rest.WriteErrorResponse(w, http.StatusNotFound, err.Error())
+			return
+		}
+
+		var scriptInfo types.ScriptInfo
+		err = cliCtx.Codec.UnmarshalJSON(res, &scriptInfo)
+		if err != nil {
+			rest.WriteErrorResponse(w, http.StatusNotFound, err.Error())
+			return
+		}
+
+		// Find TxHash and height that of transaction
+		// TODO: Get latest store tx as tx hash (wait tendermint release get result in desc order)
```

Comment:
I'm not sure I understand this TODO. We are querying for only one transaction?

Reply:
In case there are tx that store the same codehash

Reply:
Make sense. In the future when a transaction is added, then deleted, then added again.